src/llvm/enums.cr

```diff
@@ -1,34 +1,182 @@
 module LLVM
-  @[Flags]
-  enum Attribute : UInt32
-    ZExt            = 1 << 0
-    SExt            = 1 << 1
-    NoReturn        = 1 << 2
-    InReg           = 1 << 3
-    StructRet       = 1 << 4
-    NoUnwind        = 1 << 5
-    NoAlias         = 1 << 6
-    ByVal           = 1 << 7
-    Nest            = 1 << 8
-    ReadNone        = 1 << 9
-    ReadOnly        = 1 << 10
-    NoInline        = 1 << 11
-    AlwaysInline    = 1 << 12
-    OptimizeForSize = 1 << 13
-    StackProtect    = 1 << 14
-    StackProtectReq = 1 << 15
-    Alignment       = 31 << 16
-    NoCapture       = 1 << 21
-    NoRedZone       = 1 << 22
-    NoImplicitFloat = 1 << 23
-    Naked           = 1 << 24
-    InlineHint      = 1 << 25
-    StackAlignment  = 7 << 26
-    ReturnsTwice    = 1 << 29
-    UWTable         = 1 << 30
-    NonLazyBind     = 1 << 31
-    # AddressSafety = 1_u64 << 32,
-    # StackProtectStrong = 1_u64 << 33
+  {% if LibLLVM.has_constant?(:AttributeRef) %}
+    @[Flags]
+    enum Attribute : UInt64
```

Comment:
I would keep the flags, returning arrays would be very slow/expensive, I think.

In any case, I'd like to slowly remove support for older LLVM versions, but first we need to make the omnibus build work with newer LLVM versions, preferably 3.9 now.